runtime/auth/aws-signing-tests/jvm/src/aws/smithy/kotlin/runtime/auth/awssigning/tests/SigningSuiteTestBaseJVM.kt

```diff
@@ -7,7 +7,7 @@ package aws.smithy.kotlin.runtime.auth.awssigning.tests
 import aws.smithy.kotlin.runtime.auth.awscredentials.Credentials
 import aws.smithy.kotlin.runtime.auth.awscredentials.CredentialsProvider
 import aws.smithy.kotlin.runtime.auth.awssigning.*
-import aws.smithy.kotlin.runtime.auth.awssigning.middleware.AwsHttpSigner
+import aws.smithy.kotlin.runtime.auth.awssigning.AwsHttpSigner
```

Comment:
**Nit**: Unnecessary named import after star import.

Reply:
surprised ktlint didn't catch that